index.html

```diff
@@ -58,12 +58,12 @@
             </nav>
           </div>
         </div>
-        <ul class="mobile-menu">
+        <ul class="mobile-menu" id="dh">
           <li>
-            <button type="button" class="mobile-menu-disappear-btn">X</button>
+            <button type="button" class="mobile-menu-disappear-btn" id="close">X</button>
           </li>
           <li>
-            <a href="#About" class="menu-about menu-span">About</a>
+            <a href="../About.html" class="menu-about menu-span">About</a>
```

Comment:
Great design @godybuoy 🤝

- [ ] Your design is lacking an about section both for mobile and desktop views. As you can see you are trying to link to the `About.html` file but this file does not exist in your file structure. Kindly fix this and resubmit your work for review.